tests/rmq/test_process_control.py

```diff
@@ -7,45 +7,46 @@
 from kiwipy import rmq
 
 import plumpy
-import plumpy.communications
-from plumpy import process_comms
+from plumpy.rmq import process_control
 
+from . import RmqCoordinator
 from .. import utils
 
 
 @pytest.fixture
-def thread_communicator():
+def _coordinator():
     message_exchange = f'{__file__}.{shortuuid.uuid()}'
     task_exchange = f'{__file__}.{shortuuid.uuid()}'
     task_queue = f'{__file__}.{shortuuid.uuid()}'
 
-    communicator = rmq.RmqThreadCommunicator.connect(
+    comm = rmq.RmqThreadCommunicator.connect(
         connection_params={'url': 'amqp://guest:guest@localhost:5672/'},
         message_exchange=message_exchange,
         task_exchange=task_exchange,
         task_queue=task_queue,
     )
-    communicator._loop.set_debug(True)
+    comm._loop.set_debug(True)
+    coordinator = RmqCoordinator(comm)
```

Comment:
This is a typical example how I wrap the original rmq communicator into the Coordinator interface.